doc/Roadmap.md

```diff
@@ -47,7 +47,22 @@ Please contact us and tell us about what features and improvements do you see as
 * Basic one-to-one VoIP (NAT traversal and media relaying with ICE/STUN/TURN server)
 * Contributions to ecosystem
 
-## MongooseIM 3.0.x: chatbots and IoT (2017S2)
+## MongooseIM 3.0.x: real-time planetary scale (2018Q1)
+
+* Cluster intercontinental geo-distribution for real-time content
+* SASL 2.0?
+* [XEP-0376: Pubsub Account Management](https://xmpp.org/extensions/xep-0376.html)?
+* Contributions to ecosystem
+
+## MongooseIM 3.1.x:  (2018Q2)
+
+* Intercontinental geo-distribution for message archive
+* Inbox
+* [XEP-0156: Discovering Alternative XMPP Connection Methods](http://xmpp.org/extensions/xep-0156.html)?
```

Comment:
why the question marks? It's a roadmap, we know these are ideas. It looks like upspeak